indicator/messagingmenu.cpp

```diff
@@ -391,17 +391,12 @@ void MessagingMenu::addCall(const QString &targetId, const QString &accountId, c
     QContactFetchRequest *request = new QContactFetchRequest(this);
     request->setFilter(QContactPhoneNumber::match(targetId));
 
-    //FIXME: on arm64 the connect() statement below fails at runtime with the following output:
-    //QObject::connect: signal not found in QtContacts::QContactFetchRequest
-    // so we just disable it
-#ifndef __aarch64__
```

Comment:
Well this is what i call a ultra hack! damn i hope there is not more of these.. :disappointed: